visualizer-server/src/main.rs

```diff
@@ -1,5 +1,9 @@
-#[actix_web::main]
-async fn main() -> Result<(), std::io::Error> {
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
     env_logger::init();
-    visualizer_server::run::http_server(8050).await
+    let http_server = visualizer_server::run::http_server(8050);
```

Comment:
let's share the service impl (because it could contain shared state etc)
```rust
let service = Arc::new(SolidityVisualizerService::default());
let http_server = http_server(service, port);
let grpc_server = grpc_server(service, port);
```